src/models/account.js

```diff
@@ -0,0 +1,37 @@
+import mongoose from "mongoose";
+
+const accountSchema = mongoose.Schema(
+  {
+    name: {
+      type: String,
+      uppercase: true,
```

Comment:
@raiandexter0607 set as **unique** this field

Reply:
Sure, this was solved.